### PR TITLE
Updated vitals measurement best practices post with guidance on avoiding averages.

### PR DESCRIPTION
**Pull request by malchata**
Changes proposed in this pull request:

- Updates the [Best practices for measuring Web Vitals in the field post](https://web.dev/vitals-field-measurement-best-practices/) with guidance to avoid calculating averages.
- Also adds a small aside in the distributions section to advise looking at quantiles higher than the 75th.

### CODE REVIEW

**Comment by malchata** [comment]
@philipwalton, I'm updating this per your recommendation on the in-flight post for core web vitals workflows.

**Review by rachelandrew — APPROVED** [review]

**Comment by malchata** [comment]
@rachelandrew, thanks for your approval on this.

@philipwalton, I'll hold off on merging this until after you look it over.

File: src/site/content/en/blog/vitals-field-measurement-best-practices/index.md
Comment (philipwalton) [inline src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:76]: The 4-space indent here is actually per Google JS style. Any line continuation that's not a block indentation (like a object or array literal) should be indented 4 spaces instead of 2.

File: src/site/content/en/blog/vitals-field-measurement-best-practices/index.md
Comment (philipwalton) [inline src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:148]: Just curious if there's a reason you favor not indenting here.

Comment (malchata) [reply src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:148]: No so much a preference here, just didn't know!

Comment (philipwalton) [reply src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:148]: Gotcha. I don't have a preference either; I was curious to know if it was a web.dev style thing I was unaware of.

File: src/site/content/en/blog/vitals-field-measurement-best-practices/index.md
Comment (malchata) [reply src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:76]: I figure Prettier must have done something here. My editor's been kind of goofy lately with picking up config files for stuff like eslint or prettier.

Comment (malchata) [inline src/site/content/en/blog/vitals-field-measurement-best-practices/index.md:76]:
```suggestion
      fetch('/analytics', {body, method: 'POST', keepalive: true});
```

**Comment by malchata** [comment]
@philipwalton, we already have one approval from Rachel, but waiting on yours before I merge it so I don't push anything you're not OK with to the live site.

**Review by philipwalton — APPROVED** [review]

**Pull request merged**
